cobalt/media/base/sbplayer_pipeline.cc

```diff
@@ -97,6 +102,25 @@ TimeDelta AdjustWriteDurationForPlaybackRate(TimeDelta write_duration,
   return write_duration * playback_rate;
 }
 
+// The function returns the predefined frames per DecoderBuffer.
+//
+// The number of frames is used to estimate the number of sample writes for
```

Comment:
number of sample writes => number of samples per write

Reply:
Done.